app/2.0.x/health-checks-circuit-breakers.md

```diff
@@ -221,6 +221,10 @@ be re-enabled again by the system administrator.
 
 * Active health checks can automatically re-enable a target in the
 ring balancer as soon as it is healthy again. Passive health checks cannot.
+* Passive health checks generally respond faster, because there are more
+requests than probes (an active check configured with a failure threshold
+of 3 and a probe interval of 3, will take anywhere between 10 and 15 seconds
+to respond).
```

Comment:
```suggestion
* Passive health checks generally respond faster, because there are more
requests than probes. For example, an active check configured with a failure threshold
of 3 and a probe interval of 3 will take anywhere between 10 and 15 seconds
to respond.
```